packages/local_auth/local_auth_android/android/src/main/java/io/flutter/plugins/localauth/AuthenticationHelper.java

```diff
@@ -129,39 +129,51 @@ private void stop() {
   @SuppressLint("SwitchIntDef")
   @Override
   public void onAuthenticationError(int errorCode, CharSequence errString) {
+    final AuthenticationErrorHandler errorHandler = new AuthenticationErrorHandler();
+    final Runnable onStopCallback = new Runnable() {
```

Comment:
Since we moved part of error handling logic to AuthenticationErrorHandler class we need this callback to call stop() method of the AuthenticationHelper.

Reply:
Gotcha. Thanks!